### PR TITLE
Add Welcome bot

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Add Welcome bot

### CODE REVIEW

**Comment by bryceml** [comment]
can we put links to all the resources mentioned?

```
   Community Wiki
    Salt’s Contributor Guide
    Join our Community Slack
    IRC on Freenode
    SaltStack YouTube channel
    SaltStackInc Twitch channel```

**Comment by krionbsd** [comment]
> can we put links to all the resources mentioned?
> 
> ```
>    Community Wiki
>     Salt’s Contributor Guide
>     Join our Community Slack
>     IRC on Freenode
>     SaltStack YouTube channel
>     SaltStackInc Twitch channel```
> ```

yeah, forgot it, done

**Review by bryceml — DISMISSED** [review]

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have
+  a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as
+  possible. In the meantime, here’s some information that may help as you continue your Salt
+  journey.
+
+  Please be sure to review our Code of Conduct. Also, check out some of our community
```
Comment (ScriptAutomate) [inline .github/config.yml:10]:
```suggestion
  Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community
```

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  look forward to doing awesome things with you!
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have
+  a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as
+  possible. In the meantime, here’s some information that may help as you continue your Salt
+  journey.
+
+  Please be sure to review our Code of Conduct. Also, check out some of our community
```
Comment (ScriptAutomate) [inline .github/config.yml:35]:
```suggestion
  Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community
```

**Review by ScriptAutomate — CHANGES_REQUESTED** [review]
Some suggestions on adding links to our Code of Conduct.

I'm also wondering whether we should need anything in here about the Working Groups? New contributors may not know of them. Maybe an additional link like:

```
- [SaltStack Working Groups](https://github.com/saltstack/community)
```

Rendering to:
- [SaltStack Working Groups](https://github.com/saltstack/community)

Though, I do think we should review and update the community repository `README.md` file, along with information about the `docs-hub` Documentation Working Group.

_Tagging @cassandrafaris and @sagetherage for input^_

**Review by DmitryKuzmenko — APPROVED** [review]
Probably it'd be good to add a direct link to Cassandra's profile in SaltCommunity Slack. But what Slack gives to me https://saltstackcommunity.slack.com/team/UQP614KNG doesn't work in my environment.

**Comment by ScriptAutomate** [comment]
> Probably it'd be good to add a direct link to Cassandra's profile in SaltCommunity Slack. But what Slack gives to me https://saltstackcommunity.slack.com/team/UQP614KNG doesn't work in my environment.

The link to her profile properly opens when I open that link in Slack, for me.

**Review by bryceml — APPROVED** [review]

**Comment by sagetherage** [comment]
> Some suggestions on adding links to our Code of Conduct.
> 
> I'm also wondering whether we should need anything in here about the Working Groups? New contributors may not know of them. Maybe an additional link like:
> 
> ```
> - [SaltStack Working Groups](https://github.com/saltstack/community)
> ```
> 
> Rendering to:
> 
>     * [SaltStack Working Groups](https://github.com/saltstack/community)
> 
> 
> Though, I do think we should review and update the community repository `README.md` file, along with information about the `docs-hub` Documentation Working Group.
> 
> _Tagging @cassandrafaris and @sagetherage for input^_

That is fine, those can be separate PRs in those repositories, correct? I can create issues for work to be done, but I cannot do that work this week, perhaps in 2 or 3 weeks.

**Comment by krionbsd** [comment]
> https://saltstackcommunity.slack.com/team/UQP614KNG

doesn't work for me in browser, I think providing the email makes more sense though, as not everyone is ready to join the slack to ask the questions

**Review by s0undt3ch — APPROVED** [review]

**Comment by krionbsd** [comment]
re-run full windows

**Pull request merged**